docs/src/template.js

```diff
@@ -18,8 +18,26 @@
 
 import './static/favicon.ico';
 import './static/social-preview.png';
+
 import appleTouchIcon from './static/apple-touch-icon.png';
 
+/*
+PR builds get deployed to /prs/<id>/. This ensures that an environment variable
+can be set when that happens to allow routes to keep working.
+*/
+const BASE_PATH = process.env.BASE_PATH ? `/${process.env.BASE_PATH}` : '';
+
+/*
+Doesn't use tokens because it's not a component, just raw HTML.
+*/
+const warningMessage = process.env.BASE_PATH
+  ? `
+<div style="padding: 4px 8px; background-color: #ffb54d; color: #02122c; text-align: center;">
+  <strong>Pull request build</strong> — client-side routing doesn't work correctly. To navigate, use the search field.
+</div>
+`
+  : '';
```

Comment:
This should be done in react IMO.
This would be a good place to stick it in every page https://github.com/Skyscanner/backpack-docs/blob/master/docs%2Fsrc%2Findex.js#L64

Reply:
Yeah fair point, will do